apps/omg_eth/lib/eth.ex

```diff
@@ -175,7 +175,7 @@ defmodule OMG.Eth do
     end
   end
 
-  defp event_topic_for_signature(signature) do
+  def event_topic_for_signature(signature) do
```

Comment:
why is this public?

Reply:
It shouldn't be, this code is not for review yet.

Reply:
@InoMurko Fixes are uploaded, feel free to have a look.